src/core/generator/graphql_type.rs

```diff
@@ -243,17 +258,20 @@ mod tests {
     fn test_from_method() {
         let input: Vec<TestParams> = vec![
             // Methods
-            ((Entity::Method, &[], "foo"), "foo"),
-            ((Entity::Method, &["a.b.c"], "fooBar"), "a__b__c__fooBar"),
+            ((Entity::Method, &[], "foo"), "GEN__foo"),
+            (
+                (Entity::Method, &["a.b.c"], "fooBar"),
+                "GEN__a__b__c__fooBar",
+            ),
             (
                 (Entity::Method, &["a.b", "c"], "foo_bar"),
-                "a__b__c__foo_bar",
+                "GEN__a__b__c__foo_bar",
             ),
-            ((Entity::Method, &[], "a.b.c.foo"), "a_b_c_foo"),
-            ((Entity::Method, &["a.b.c"], "foo"), "a__b__c__foo"),
-            ((Entity::Method, &["a.b"], "d.e.foo"), "a__b__d_e_foo"),
-            ((Entity::Method, &[""], "a.b.c.foo"), "a_b_c_foo"),
-            ((Entity::Method, &[], "a_b_c_foo"), "a_b_c_foo"),
+            ((Entity::Method, &[], "a.b.c.foo"), "GEN__a_b_c_foo"),
+            ((Entity::Method, &["a.b.c"], "foo"), "GEN__a__b__c__foo"),
+            ((Entity::Method, &["a.b"], "d.e.foo"), "GEN__a__b__d_e_foo"),
+            ((Entity::Method, &[""], "a.b.c.foo"), "GEN__a_b_c_foo"),
+            ((Entity::Method, &[], "a_b_c_foo"), "GEN__a_b_c_foo"),
```

Comment:
I think the prefix can be moved to `assert_type_names`

Reply:
done.